roadmap/BASIC_TYPES.md

```diff
@@ -1,9 +1,9 @@
 # Basic types
 
 
-Because Elm and Elixir share a lot of common basic types, there is no need to redefine all of them for Elchemy need. For simplicity and interoperability some of the standard types translate directly to each other.
+Because Elm and Elixir share a lot of common basic types, there is no need to redefine all of them for Elchemy. For simplicity and interoperability some of the standard types translate directly to each other.
 
-Here is a table of all standard types used in Elchemy environment and their Elixir equivalents:
+Here is a table of all standard types used in the Elchemy environment and their Elixir equivalents:
```

Comment:
Are you sure about that?
I was always convinced that articles work based on the next word, rather than a word they apply to (so article based on not using "the" before `Elchemy`, rather than using "the" for the `environment`)

Reply:
A friend confirmed it, while I honestly do this by intuition, the exact rule for this is also largely out of my focus. This here might help to bring light into the thing: https://owl.english.purdue.edu/owl/resource/540/01/

Reply:
Fair enough 
I'm convinced and I'm definitely not in a position to judge anyway 👍

Reply:
`the` is correct in that position.  :-)

Reply:
You are everywhere, arent you? :D

Reply:
@ShalokShalom Oh yes he is 😄